prov/rxm/src/rxm_conn.c

```diff
@@ -199,6 +199,20 @@ static void rxm_conn_handle_eq_err(struct rxm_ep *rxm_ep, ssize_t rd)
 	}
 }
 
+static void rxm_conn_handle_postponed_op(struct rxm_ep *rxm_ep,
+					 struct util_cmap_handle *handle)
+{
+	struct rxm_tx_entry *tx_entry;
+	struct rxm_conn *rxm_conn = container_of(handle, struct rxm_conn, handle);
+
+	while (!dlist_empty(&rxm_conn->postponed_tx_list)) {
+		dlist_pop_front(&rxm_conn->postponed_tx_list, struct rxm_tx_entry,
+				tx_entry, postponed_entry);
+		if (!(tx_entry->comp_flags & FI_RMA))
```

Comment:
If this check isn't always true, then we're leaking the postponed entries being popped off the list.

Reply:
This check is always true at the moment. The handling of RMA tx entries has been added in the PR #3822